### PR TITLE
2431 fix inline form properties

### PR DESCRIPTION
**Pull request by AJax2012**
- [x] Bug fix 
- [ ] New feature 
- [ ] Chore 
- [ ] Breaking change 
- [x] There is an open issue which this change addresses
- [x] I have read the **[CONTRIBUTING](./CONTRIBUTING.md)** document.
- [x] My commits follow the [Git Commit Guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commits)
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly.
- [x] My change requires a change to [Typescript typings](./types/lib).
  - [x] I have updated the typings accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

This is related to issue #2431 and replaces PR #2568. 

.form-inline class no longer exists in bootstrap and needs to be replaced. The property inline in form label hasn't existed in reactstrap since 2017 (commit e9b780350ca428f3da7c609cd8902dfb226924ee) and needed to be removed from the types file. I have updated all dependencies in the project to reflect the changes and had to remove 1 test (that was testing that the inline property added the .form-inline class to the component).

Note: I'm not sure how to test if this would be a "breaking change", but the class doesn't work, so I assume this wouldn't break anything? Please let me know if this needs to be changed. Thanks!

### CODE REVIEW

**Review by illiteratewriter — APPROVED** [review]

**Comment by davidacevedo** [comment]
Seems like this needs some rebasing

**Comment by AJax2012** [comment]
@davidacevedo sorry for taking so long to respond - I've been incredibly busy the past few weeks... I went through and looked at the guidelines again and I'm not sure who is responsible for this or how to request an approval before code becomes outdated again.

If I go through and make the required rebase changes, will my PR sit until it needs to be rebased again? Is there a way to get in contact with someone to get these changes approved before there's another merge conflict?

**Comment by illiteratewriter** [comment]
@AJax2012 we'll try and make sure that this PR won't sit for long.

**Comment by AJax2012** [comment]
> @AJax2012 we'll try and make sure that this PR won't sit for long.

Thanks! Sorry if I came across harsh; I just didn't want to get caught in an infinite loop, haha. I'll try to get these changes made after work today.

**Comment by AJax2012** [comment]
@illiteratewriter I think it should be all good to go. Let me know if there's anything that needs to be done. Thanks again!

File: src/Form.js [hunk 1]
```diff
@@ -43,15 +41,15 @@ class Form extends Component {
...
     } = this.props;
 
-    const classes = mapToCssModules(
-      classNames(className, inline ? 'form-inline' : false),
-      cssModule,
+    const classes = mapToCssModules(className, cssModule);
+
+    return (
+      <Tag {...attributes} ref={innerRef} className={classes} />
     );
```
Comment (illiteratewriter) [inline src/Form.js:54]: Multiple return statements doing the same thing.

**Comment by illiteratewriter** [comment]
There are a few additional files, ModalBackdrop.js, ModalDestructuring.js and ModalFocusAfterClose.js. Did something break while merging master?

File: src/Form.js (hunk 1 above)
Comment (AJax2012) [reply src/Form.js:54]: could have sworn I resolved this yesterday... I'll get it fixed. Sorry!

**Comment by AJax2012** [comment]
did a rebuild and a re-install and it worked... Ah well. Anyway, I went through and triple checked this time; as far as the merge conflicts and the updates I wanted to make go, everything seems to be correct now. Sorry for the confusion earlier!

**Comment by illiteratewriter** [comment]
@AJax2012 can you run `yarn prettier` to format the code?

**Comment by AJax2012** [comment]
> @AJax2012 can you run `yarn prettier` to format the code?

Certainly. Thought it would do that on save on my machine, but it apparently missed a file... Anyway, should be updated now.

**Comment by illiteratewriter** [comment]
@AJax2012 Thank you for the PR. :)

**Comment by AJax2012** [comment]
@illiteratewriter Thank you for helping me get everything in order and merging it!

**Pull request merged**